src/site/topic23.rst

```diff
@@ -189,6 +189,64 @@ Add
 Min & Remove Min
 ----------------
 
+* Given the nature of the binary search tree ordering, the node with the minimum element **must** be in the leftmost node in the tree
+* There are three cases we'll have to take care of when looking for the minimum node
+    1. The leftmost node is the root (root has no left child)
+    2. The leftmost is a leaf node (no children)
+    3. The leftmost node is an interior node
+
+* Finding the minimum is simple
+    * Assuming a root node exists, keep going to the left subtree until there is no more left subtree
+
+* Removing will be a little trickier since we may be removing something that has children that must remain in the tree
+* And further, not only must the children stay in the tree, but the proper binary search tree ordering **must** be preserved
+
+* Consider the following examples and think of how one would need to manage potential children of a node being removed
+    * For this example, assume duplicate values will go to the right
```

Comment:
This doesn't matter. If duplicates went left, we'd remove the left-most one.

Reply:
Only added this since I said the following later

> Everything to the right of the root is larger than (or equal to) the value in the root

Reply:
It still doesn't matter. If duplicates go left, we're removing the left-most node. If duplicates go right, we're removing the left-most node. The later statement is still accurate regardless of which way duplicates go.